content/id/docs/tasks/tools/kubeadm/high-availability.md

```diff
@@ -0,0 +1,364 @@
+
+
+
+<!-- steps -->
+
+## Langkah pertama untuk kedua metode
+
+### Membuat _load balancer_ untuk kube-apiserver
+
+{{< note >}}
+Akan ada banyak konfigurasi untuk _load balancer_. Contoh berikut ini hanyalah salah satu
+opsi. Kebutuhan klastermu mungkin membutuhkan konfigurasi berbeda.
+{{< /note >}}
+
+1.  Buat sebuah _load balancer_ kube-apiserver dengan sebuah nama yang yang akan mengubah ke dalam bentuk DNS.
+
+    - Pada _environment cloud_ kamu harus meletakkan Node _control plane_ di belakang _load balancer_ yang meneruskan TCP. _Load balancer_ ini mendistribusikan trafik ke seluruh Node _control plane_ pada daftar tujuan. _Health check_ untuk
+      apiserver adalah pengujian TCP pada porta yang didengarkan oleh kube-apiserver
+      (nilai semula `:6443`).
+
+    - Tidak direkomendasikan untuk menggunakan alamat IP secara langsung pada _environment cloud_.
+
+    - _Load balancer_ harus dapat berkomunikasi dengan seluruh Node _control plane_ 
+      pada porta yang digunakan apiserver. _Load balancer_ tersebut juga harus mengizinkan trafik masuk pada porta yang didengarkannya.
+
+    - Pastikan alamat _load balancer_ sesuai
+      dengan alamat `ControlPlaneEndpoint` pada kubeadm.
+
+    - Baca panduan [Opsi untuk _Software Load Balancing_](https://github.com/kubernetes/kubeadm/blob/master/id/docs/ha-considerations.md#options-for-software-load-balancing)
+      untuk detail lebih lanjut.
+
+2.  Tambahkan Node _control plane_ pertama pada _load balancer_ dan lakukan pengujian koneksi:
+
+    ```sh
+    nc -v LOAD_BALANCER_IP PORT
+    ```
+
+    - Kegalatan koneksi yang ditolak memang diantisipasi karena apiserver belum
+      berjalan. Namun jika mendapat _timeout_, berarti _load balancer_ tidak dapat berkomunikasi
+      dengan Node _control plane_. Jika terjadi _timeout_, lakukan pengaturan ulang pada _load balancer_ agar dapat berkomunikasi dengan Node _control plane_.
+
+3.  Tambahkan Node _control plane_ lainnya pada grup tujuan _load balancer_.
+
+## Node _control plane_ dand etcd bertumpuk (_stacked_)
+
+### Langkah-langkah untuk Node _control plane_ pertama
+
+1.  Inisialisasi _control plane_:
+
+    ```sh
+    sudo kubeadm init --control-plane-endpoint "LOAD_BALANCER_DNS:LOAD_BALANCER_PORT" --upload-certs
+    ```
+
+    - Kamu bisa menggunakan opsi `--kubernetes-version` untuk mengatur versi Kubernetes yang akan digunakan.
+      Direkomendasikan untuk menggunakan versi kubeadm, kubelet, kubectl, dan Kubernetes yang sama.
+
+    - Opsi `--control-plane-endpoint` harus diatur menuju alamat atau DNS dan porta dari _load balancer_.
+
+    - Opsi `--upload-certs` digunakan untuk mengunggah sertifikat-sertifikat yang harus dibagikan ke seluruh
+      Node _control-plane_ pada klaster. Jika sebaliknya, kamu memilih untuk menyalin sertifikat ke
+      seluruh Node _control-plane_ sendiri atau menggunakan perkakas automasi, silakan hapus opsi ini dan merujuk ke bagian [Distribusi sertifikat manual](#manual-certs) di bawah.
```

Comment:
if you decide to remove the alias anchor here (https://github.com/kubernetes/website/pull/22669/files#diff-6f6c6f2a2b1a160e3bf2cac7899fe8c6R282), this needs change

Reply:
Removed the alias and updated the anchor in bc5c6c5.